benchmarks/Main.hs

```diff
@@ -87,12 +87,13 @@ recallPrefixed65535 =
 
 formatExponential :: Int -> Double -> String
 formatExponential precision d =
-    show (roundToPrec sig10) ++ "e" ++ show exp10Int
+  show (roundToPrec sig10) ++ "e" ++ show (exp10Int :: Int)
```

Comment:
I would put this `:: Int` type annotation with the definition below, not inline here.

Sorry, missed this on the first review.